### DISCUSSION TITLE
由于字体原因编译或使用报错的，建议安装2021版本的texlive

### DISCUSSION THREAD

**Discussion by 2foil** [nuaatug/nuaathesis#98] [General]
如题，最近在使用nuaathesis模板，mac和win下面一直报错，翻阅issues发现和newtx以及newpx在2022年的更新有关，
果断将mactex（mac）和texlive（win）回退到2021版，重试了一下，目前mac下和win下都丝滑编译和使用。
因此如果遇到字体报错用不了的，建议回退texlive2021版本

相关issues：
#83 #85 #88

**Comment by yzwduck** [comment]
(不建议退回旧版本, 照着上面 issue 提到的在 global.tex 里添加 `\defaultfontfeatures{}` 就能编译了. 如果有新版本发布的话, 这句话会被加到模板里.)

**Reply by 2foil** [reply]
尝试了之前issues中提到的解决方法，还是不能成功编译/使用，mac和win下依旧报错。
以及另一个问题就是目录很引文索引无法显示 https://github.com/nuaatug/nuaathesis/issues/88#issuecomment-1247459417

😂回退的解决方法当成一个参考吧，如果铁了心毕业论文要用LaTex，并且无法正常使用/编译的话，可以试试这种方法。
